### ISSUE TITLE
[suggestion] Clarify the choose-blueprint-updates step 

### ISSUE THREAD

**Issue by ijlee2** [ember-cli/ember-cli-update#1109]
## Description

Hello. I was helping out a couple of people on Discord.

- https://discord.com/channels/480462759797063690/486548111221719040/788191907943284737
- https://discord.com/channels/480462759797063690/483601670685720591/816209083623145472

They weren't sure what to do when they encountered the following prompt in an initial step of the `ember-cli-update` wizard.

![Initial step of choose blueprint updates](https://user-images.githubusercontent.com/16869656/109615013-a95e2500-7b33-11eb-9d9f-8d2246699448.png)

I think this happened due to a couple of factors:

- A user didn't want to upgrade to the latest version (3.25), but to some prior version as a stepping stone (3.16, in the example above).
- The prompt doesn't specify what key to press to continue (the prompt only mentions arrow keys).

## Suggested solution

I looked at https://github.com/ember-cli/ember-cli-update/blob/master/src/choose-blueprint-updates.js to see how the blueprint update options were created.

I think we could either:

- Update the prompt to clarify that the user can set the target update version in the next step
- Update the blueprint update options to hide `latest`, or include the user's `--to` value
- Look into another solution

Can you let us know what you think about this suggestion? Thanks!

**Comment by sly7-7** [comment]
I was a bit confused as well by this prompt. I would go for `Update the blueprint update options to hide latest, or include the user's --to value`

**Comment by kellyselden** [comment]
I think the docs need to be updated. `--to` is being phased out. This is because there can be multiple blueprints being tracked in ember-cli-update.json. Because of this, reusing `--to` might not work based on which blueprint you pick.

**Comment by sly7-7** [comment]
@kellyselden Thanks for the feedback. So you're suggesting to remove the `--to` example from the readme ?
That mean that running `ember-cli-update` will then prompt the user to choose the targeted version ?

**Comment by kellyselden** [comment]
Correct. The workflow is you first pick a blueprint (most commonly only a single one), then you pick what will become the --to.

Now that I think of it, maybe if there's only a single blueprint and they've supplied a --to, we can skip the blueprint prompt?

**Comment by sly7-7** [comment]
It could be a good move.
Though, I've just tested without `--to` and it already works as this I think.
![Screenshot from 2021-03-04 19-52-25](https://user-images.githubusercontent.com/1826661/110014526-2dc2ca80-7d23-11eb-9523-e5c568cce46b.png)
Not related to this issue, bu if possible, I think this could be a very good thing to suggest the available versions if choosing semver
